templates/artist/includes/artist_detail_updates.html

```diff
@@ -1,6 +1,7 @@
 <div class="tabs-panel {% if form.errors %}is-active{% endif %}" id="updates">
     {% if updates %}
-        <h2>UPDATES</h2>
+        <h2>ARTIST UPDATES</h2>
+        <h4>HERE ARE UPDATES FROM YOUR ARTISTS</h4>
```

Comment:
We should revert this change here, since this is on the artist detail page and visible to everyone.